yt_idefix/definitions.py

```diff
@@ -0,0 +1,29 @@
+# Physical constants in c.g.s units defined in $PLUTO_DIR/Src/pluto.h
+pluto_def_constants = {
```

Comment:
I'll probably need to update this in the future if upstream (Pluto) changes a value or add new ones. Could you please document (in a comment) which version of Pluto you used as a reference here ?

Reply:
Sure